### ISSUE TITLE
A11y_VscodeWebviewUiToolkit_Breadcrumb+BreadcrumbItem_ColorContrast: The color contrast ratio of link texts of "src", "components", "index" are less than 4.5:1

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode-webview-ui-toolkit#177]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_VscodeWebviewUiToolkit_Aug2021;#Webview UI Toolkit for Visual Studio Code;#Win32;#FTP;#DesktopApp;#WCAG1.4.3;#AINotInScope;#ColorContrast;

### Environment Details:
Application Name: Vscode Webview Ui Toolkit
Windows Version: Win10

### Repro Steps:

1. Launch VS Code
2. Open WebView UI Toolkit: All Component extension
3. Tab to Breadcrumb+Breadcrumb Item
4. Open Color contrast analyzer and check if color contrast ratio of link texts of "src", "components", "index" are less than 4.5:1. 

### Actual:
The color contrast ratio of link texts of "src", "components", "index" are less than 4.5:1.It is 3.895:1

### Expected:
The color contrast ratio of link texts of "src", "components", "index" should be greater than or equal to 4.5:1

### User Impact:
Low vision users will find it difficult to view the links.

**Comment by fsteffi** [comment]
**Githubtags**:#Closed;
